synapsemonitor/__main__.py

```diff
@@ -24,7 +24,8 @@ def monitor_cli(syn, args):
         syn_id=args.synapse_id,
         email_subject=args.email_subject,
         users=args.users,
-        days=args.days,
+        value=args.value,
+        unit=args.unit,
```

Comment:
`value` and `unit` is sort of confusing as variable names.  Value seems too broad.  Not sure if I have suggestions - let me look at how other packages do this.

Reply:
Could we try using this: https://stackoverflow.com/a/68686625

Reply:
I agree it is not super intuitive.  But I actually took the `value` and `unit` terminology directly from the rate expressions on AWS batch: `rate(value unit)` 

https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html

I guess it depends on whether we want to keep synapsemonitor timing consistent with timing for scheduled jobs

Reply:
@thomasyu888 Given the utility of direct integration with scheduled jobs in the service catalog, what do you think about retaining the consistent input here as is for specifying time value and time unit?

Reply:
I'm still on the fence about it, just because it's not really that intuitive at first glance.  It is documented pretty well though.

Im tempted to just support `hours` since it is the lowest value, but it is more convenient having both `unit` and `value`.   I also thought about creating a `Rate` class with just `unit` and `value`, but that seems overkill if people have to define that class every time they want to use the function.

Reply:
To make it more intuitive, I could modify the input parameter to group the value and unit together, `--rate value unit`, with the default still being `--rate 1 day`, and then add custom checks for the two inputs.

```
parser_monitor.add_argument(
        "--rate",
        "-r",
        nargs=2,
        metavar=("value", "unit"),
        default=(1, "day"),
        help="Find modifications to File entities in the last {value} {unit}. "
        "(default: %(default)s)",
    )
```
looking like
```
usage: synapsemonitor monitor [-h] [--users USERS [USERS ...]] [--output OUTPUT] [--email_subject EMAIL_SUBJECT] [--rate value unit]
                              synapse_id

positional arguments:
  synapse_id            Synapse ID of entity to be monitored.

optional arguments:
  -h, --help            show this help message and exit
  --users USERS [USERS ...], -u USERS [USERS ...]
                        User Id or username of individuals to send report. If not specified, defaults to logged in Synapse user.
  --output OUTPUT, -o OUTPUT
                        Output modified entities into this csv file. (default: None)
  --email_subject EMAIL_SUBJECT, -e EMAIL_SUBJECT
                        Sets the subject heading of the email sent out. (default: New Synapse Files)
  --rate value unit, -r value unit
                        Find modifications to File entities in the last {value} {unit}. (default: (1, 'day'))
```

although should probably use something other than `--rate`.  `--timing` perhaps?  what do you think?

Reply:
I like this.